src/Serializer/AbstractSerializer.php

```diff
@@ -318,11 +319,7 @@ private function serializeCallableParameters(\ReflectionFunctionAbstract $reflec
                 $paramName = '[' . $paramName . ']';
             }
 
-            if ($paramType) {
-                $params[] = $paramType . ' ' . $paramName;
-            } else {
-                $params[] = $paramName;
-            }
+            $params[] = $paramType . ' ' . $paramName;
```

Comment:
Did you mean if `$paramType` is missing? Actually it cannot happen because if the parameter isn't typehinted we report the type `mixed`, so the code should be the same as before. To clarify, the `else` branch was never executed